waku/v2/protocol/waku_swap/waku_swap.nim

```diff
@@ -241,10 +241,27 @@ proc init*(wakuSwap: WakuSwap) =
     else:
       info "Payment threshhold not hit"
 
+  proc balance() : BalanceReport {.gcsafe, closure.} = 
+    
+    # Loop through the accounting state for each peer to indicate if the current peer is at a profit or loss
```

Comment:
Note that a balance is only well-defined on a pairwise basis, i.e. there's no real "total debit" or "total credit" concept.

Example: you could tip a lot at one restaurant, but that doesn't mean you can buy a coffee and not pay for it

Maybe this should be a map? Can define a `$` to print function for displaying this for all peers